paradise_papers_django/search/templates/search/display_information/officer.html

```diff
@@ -1,15 +1,155 @@
-<h1 class="company-title"> {{ node_info.name }} </h1>
-<p>Linked countries: {{ node_info.countries }}</p>
-<p>Data From: {{node_info.sourceID}}</p>
+{% load nodes_extra %}
+<div class="node">
+  <header>
+      <h3 class="node-title">OFFICER</h3>
+      <h1 class="company-title">{{ node_info.name }}</h1>
+   </header>
+    <div class="main-info">
+       <div class="left-side">
+          <ul class="relationships">
+            {% if entities %}
+              <li>
+                <span>Connected to <strong>{{ entities|length }}{% if entities|length > 1 %} Entities {% else %} Entity {% endif %}</strong></span>
+              </li>
+            {% endif  %}
+            {% if officer_connections %}
+               <li>
+                  <span> Connected to <strong>{{ officer_connections|length }}{% if officer_connections|length > 1 %} Officers {% else %} Officer {% endif %}</strong></span>
+               </li>
+            {% endif %}
+             {% if others %}
+                <li>
+                   <span> Connected to <strong>{{ others|length }}{% if others|length > 1 %} Others {% else %} Other {% endif %}</strong></span>
+                </li>
+            {% endif %}
+            {% if addresses %}
+               <li>
+                   <span>Connected to <strong>{{ addresses|length }}{% if addresses|length > 1 %} Addresses {% else %} Address {% endif %}</strong></span>
+               </li>
+            {% endif  %}
+            {% if intermediaries %}
+            <li>
+               <span> Connected to <strong>{{ intermediaries|length }}{% if intermediaries|length > 1 %} Intermediaries {% else %} Intermediary {% endif %}</strong></span>
+            </li>
+            {% endif %}
 
-{% if addresses %}
-    <div>
-        <span>Connected to<strong> {{ addresses|length }} Address</strong></span>
-    </div>
-{% endif  %}
+          </ul>
+           <ul class="node-information">
+            <li>
+               <span>Linked countries:</span>
+                {% with node_info.countries|split:";" as countries %}
+                    {% for skill in countries %}
+                        <a class="node-link" href="#">{{ skill }}</a>, 
```

Comment:
Hello a quick question 😅 
Why this link doesn't redirect to another page? 🤔

Reply:
That is out of this scope, it should redirect to the lookup page with the country selected as default for the search